yml/dp03_10yr.yaml

```diff
@@ -359,8 +360,8 @@ tables:
     ivoa:ucd: meta.code
 - name: DiaSource
   "@id": "#DiaSource"
-  description: Table to store 'difference image sources' - sources detected at SNR
-    >=5 on difference images.
+  description: "Table to store 'difference image sources' - sources detected at SNR >=5 on difference images."
```

Comment:
When this is deployed, let's make sure that we verify that the nested single quotes are flowed successfully through all the way to PyVO, on the Notebook side, and to the Portal UIs.

Reply:
Okay, I've just put this in double quotes because it was messing up syntax highlighting of this file in my IDE without them. I'll make a note to check what it looks like on the Portal once deployed.